x/incentive/keeper/elys_delegator.go

```diff
@@ -0,0 +1,63 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/elys-network/elys/x/incentive/types"
+)
+
+// SetElysDelegator set a specific elysDelegator in the store from its index
+func (k Keeper) SetElysDelegator(ctx sdk.Context, elysDelegator types.ElysDelegator) {
```

Comment:
We won't use normal Cosmos SDK staking module for delegation?

Reply:
Okay noted. Will get rid of it.